test/setup/setup.ts

```diff
@@ -142,7 +141,7 @@ beforeEach(() => {
 	global.gheUserTokenNock = githubUserToken(gheApiNock);
 	global.gheAppTokenNock = githubAppToken(gheApiNock);
 	global.mockSystemTime = (time: number | string | Date) => {
-		const mock = jest.isMockFunction(Date.now) ? mocked(Date.now) : jest.spyOn(Date, "now");
+		const mock = jest.isMockFunction(Date.now) ? jest.mocked(Date.now) : jest.spyOn(Date, "now");
```

Comment:
Just highlighting this for other reviewers, `mocked` from ts-jest is deprecated and gone in the next major update 28(which is already released, we are on 27).